package.json

```diff
@@ -5,8 +5,10 @@
   "main": "app.js",
   "dependencies": {
     "express": "^4.13.3",
+    "material-ui": "^0.11.0",
     "react": "0.13.3",
-    "react-router": "^0.13.3"
+    "react-router": "^0.13.3",
+    "react-tap-event-plugin": "^0.1.7"
```

Comment:
remove the ^ from the 0.something versions to ensure that no changes break our code in the future